### PR TITLE
CLDC-2245 Add user status filter

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Add users status filter
image

### CODE REVIEW

File: app/controllers/organisations_controller.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@ class OrganisationsController < ApplicationController
   before_action :find_resource, except: %i[index new create]
   before_action :authenticate_scope!, except: [:index]
   before_action :session_filters, if: -> { current_user.support? || current_user.organisation.has_managing_agents? }, only: %i[lettings_logs sales_logs email_lettings_csv download_lettings_csv email_sales_csv download_sales_csv]
+  before_action :session_filters, only: %i[users]
```
Comment (natdeanlewissoftwire) [inline app/controllers/organisations_controller.rb:9]: Assuming this is intentionally not included with the line above

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -259,8 +259,8 @@
     let(:organisation_1) { create(:organisation, :without_dpc, name: "A") }
     let(:organisation_2) { create(:organisation, :without_dpc, name: "B") }
     let!(:user_1) { create(:user, name: "Joe Bloggs", email: "joe@example.com", organisation: organisation_1, role: "support") }
-    let!(:user_3) { create(:user, name: "Tom Smith", email: "tom@example.com", organisation: organisation_1, role: "data_provider") }
     let!(:user_2) { create(:user, name: "Jenny Ford", email: "jenny@smith.com", organisation: organisation_1, role: "data_coordinator") }
+    let!(:user_3) { create(:user, name: "Tom Smith", email: "tom@example.com", organisation: organisation_1, role: "data_provider") }
```
Comment (natdeanlewissoftwire) [inline spec/models/user_spec.rb:263]: nice

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm!

File: app/controllers/organisations_controller.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/controllers/organisations_controller.rb:9]: Yeah, it's cause the conditions for running this are different on different paths. Didn't want to clutter the above line with too much logic either

**Review by ghost — APPROVED** [review]

**Pull request merged**